runtime/drivers/duckdb/olap.go

```diff
@@ -724,6 +738,40 @@ func (c *connection) convertToEnum(ctx context.Context, table, col string) error
 	return nil
 }
 
+// duckDB raises Contents of view were altered: types don't match! error even when number of columns are same but sequence of column changes in underlying table.
+// This causes temporary query failures till the model view is not updated to reflect the new column sequence.
+// We ensure that view for external table storage is always generated using a stable order of columns of underlying table.
+// Additionally we want to keep the same order as the underlying table locally so that we can show columns in the same order as they appear in source data.
+// Using `AllowHostAccess` as proxy to check if we are running in local/cloud mode.
+func (c *connection) generateSelectQuery(ctx context.Context, db string) (string, error) {
+	if c.config.AllowHostAccess {
+		return fmt.Sprintf("SELECT * FROM %s.default", safeSQLName(db)), nil
+	}
+
+	rows, err := c.Execute(ctx, &drivers.Statement{
+		Query: fmt.Sprintf(`
+			SELECT column_name AS name
+			FROM information_schema.columns
+			WHERE table_catalog = %s AND table_name = 'default'
+			ORDER BY name ASC`, safeSQLString(db)),
+	})
+	if err != nil {
+		return "", err
+	}
+	defer rows.Close()
+
+	cols := make([]string, 0)
+	var col string
+	for rows.Next() {
+		if err := rows.Scan(&col); err != nil {
+			return "", err
+		}
+		cols = append(cols, col)
```

Comment:
Shouldn't it use `safeName` for the `col`? Otherwise, the statement will break if there's a column named e.g. `hel"lo`